recipes/rpaframework-recognition/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "rpaframework-recognition" %}
+{% set version = "0.5.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/rpaframework-recognition-{{ version }}.tar.gz
+  sha256: dd4ae959b9de7b1006636f66f28cfbe28200434eb1499b79f3924626b2e1d978
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - poetry
+    - python >=3.6,<4.0
+  run:
+    - numpy ==1.18.4
```

Comment:
is it really necessary to pin numpy so strongly?
In general, numpy is pinned by the conda-forge-pinnings which ensure consistency across the ecosystem. Whenever a pin is updated in the conda-forge pinnings, a migration is triggered. 

https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/master/recipe/conda_build_config.yaml

If you pin numpy this strongly, installing this package with other packages depending on numpy (scipy, scikit-learn ... ) might not be possible.

Reply:
Will try migrating to an ecosystem pin.

Reply:
Fixed in e15a50ceb167bc7c4466fecfe215381ea6972524